gpii/node_modules/testing/src/Mocks.js

```diff
@@ -337,7 +349,8 @@ fluid.defaults("gpii.test.integration.mockSettingsHandlerRegistry", {
         }
     },
     listeners: {
-        "onCreate.populate": "gpii.test.integration.mockSettingsHandlerRegistry.populate"
+        "onCreate.populate": "gpii.test.integration.mockSettingsHandlerRegistry.populate",
+        "onDestroy.depopulate": "gpii.test.integration.mockSettingsHandlerRegistry.depopulate"
```

Comment:
I would be nice to test this mechanism.  Can we get a minimal test that spins up the registry, pollutes it, spins down, then spins up and accesses the registry a second time, confirming that there's no stale material?

Reply:
I've instead added a static check to the builtin sequences that makes a check for any stale components on every test startup. Your amazing work on sequence grading our integration tests made this enormously easy.